projects/packages/search/src/class-plan.php

```diff
@@ -122,19 +122,33 @@ public function update_search_plan_info( $response ) {
 		$body        = json_decode( wp_remote_retrieve_body( $response ), true );
 		$status_code = wp_remote_retrieve_response_code( $response );
 
-		if ( 200 !== $status_code || ! isset( $body['supports_instant_search'] ) ) {
+		if ( 200 !== $status_code ) {
+			return null;
+		}
+
+		return $this->set_plan_options( $body );
+	}
+
+	/**
+	 * Set plan info to options table
+	 *
+	 * @param array $plan_info - the decoded plan info array.
+	 */
+	public function set_plan_options( $plan_info ) {
```

Comment:
My feedback for `update_search_plan_info` regarding the return signature applies here as well.

Reply:
fixed in c1ed321a1.